pupil_src/capture/world.py

```diff
@@ -149,9 +149,14 @@ def get_timestamp():
     default_plugins = [('Log_Display',{}),('Dummy_Gaze_Mapper',{}),('Display_Recent_Gaze',{}), ('Screen_Marker_Calibration',{}),('Recorder',{}),('Pupil_Remote',{})]
 
 
+    def should_call_ui_functions(window):
```

Comment:
gl_utils do not have access to `g_pool.iconified`. Should we implement a poll (call `glfwGetWindowAttrib `) on the `iconified` attribute as well?